webrtc/RTCTrackEvent-fire.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Change of msid in remote description should trigger related track events</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+const sdpBase =`v=0
+o=- 5511237691691746 2 IN IP4 127.0.0.1
+s=-
+t=0 0
+a=group:BUNDLE 0
+a=ice-options:trickle
+a=ice-lite
+a=msid-semantic:WMS *
+m=audio 9 UDP/TLS/RTP/SAVPF 111 103 9 102 0 8 105 13 110 113 126
+c=IN IP6 ::
+a=rtcp:9 IN IP6 ::
+a=rtcp-mux
+a=mid:0
+a=sendrecv
+a=ice-ufrag:z0i8R3C9C4hPRWls
+a=ice-pwd:O7bPpOFAqasqoidV4yxnFVbc
+a=ice-lite
+a=fingerprint:sha-256 B7:9C:0D:C9:D1:42:57:97:82:4D:F9:B7:93:75:49:C3:42:21:5A:DD:9C:B5:ED:53:53:F0:B4:C8:AE:88:7A:E7
+a=setup:actpass
+a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
+a=extmap:9 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtpmap:103 ISAC/16000
+a=rtpmap:9 G722/8000
+a=rtpmap:102 ILBC/8000
+a=rtpmap:0 PCMU/8000
+a=rtpmap:8 PCMA/8000
+a=rtpmap:105 CN/16000
+a=rtpmap:13 CN/8000
+a=rtpmap:110 telephone-event/48000
+a=rtpmap:113 telephone-event/16000
+a=rtpmap:126 telephone-event/8000`;
+
+const sdp0 = sdpBase + `
+`;
+
+const sdp1 = sdpBase + `
+a=msid:1 1
+a=ssrc:1 cname:1
+a=ssrc:1 msid:1 1
+`;
+
+async function applyRemoteDescriptionAndReturnTrackEventStreams(test, pc, sdp)
+{
+    let testTrackPromise = new Promise((resolve, reject) => {
+        pc.ontrack = (event) => {
+            resolve(event.streams ? event.streams : []);
+        }
+        test.step_timeout(() => { reject("No track event fired") }, 5000);
+    });
+    await pc.setRemoteDescription({type: 'offer', 'sdp': sdp});
+    return testTrackPromise;
+}
+
+promise_test(async test => {
+    const pc = new RTCPeerConnection();
+
+    let streams = await applyRemoteDescriptionAndReturnTrackEventStreams(test, pc, sdp0);
+
+    const answer = await pc.createAnswer();
+    await pc.setLocalDescription(answer);
+
+    streams = await applyRemoteDescriptionAndReturnTrackEventStreams(test, pc, sdp1);
+    assert_equals(streams.length, 1, "track event has a stream");
+    assert_equals(streams[0].id, "1", "msid should match");
```

Comment:
Do we wanna assert before (`0`) and after (`1`)?

Also, since this is novel, maybe assert `streams[0].getTracks()[0] == track`?